### PR TITLE
Secor v0.26 w/ Docker

### PR DESCRIPTION
**Pull request by frnidito**
Code review created only for the purpose of showing the diff in a more reasonable way, not meant for integration yet

### CODE REVIEW

**Comment by frnidito** [comment]
Run as 
```
~/dev/secor$ docker run -it -v `pwd`/config:/home/secor/config 751c8fd4f3c7
Sep 21, 2018 9:53:12 AM com.twitter.logging.Logger log
INFO: Starting LatchedStatsListener
Sep 21, 2018 9:53:12 AM com.twitter.logging.Logger log
INFO: Admin HTTP interface started on port 9998.
Sep 21, 2018 10:02:33 AM com.twitter.common.zookeeper.DistributedLockImpl unlock
INFO: Cleaning up this locks ephemeral node.
Sep 21, 2018 10:02:33 AM com.twitter.common.zookeeper.DistributedLockImpl cleanup
INFO: Cleaning up!
^C^C
~/dev/secor$
```

Produced files in https://console.cloud.google.com/storage/browser/vr-test-kafka-logs/v2/ads_raw_st/dt=2018-09-21/?project=viralize-staging&folder=true&organizationId=122454823633

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,41 @@
-FROM openjdk:8
+FROM maven:3.5-jdk-8 as BUILD
+
+# set up certificates for twitter
+RUN echo -n | openssl s_client -connect maven.twttr.com:443 | \
+    sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' > /tmp/twttrcert.crt
+RUN keytool -import -trustcacerts -keystore $JAVA_HOME/jre/lib/security/cacerts \
+    -storepass changeit -noprompt -alias twttrcert -file /tmp/twttrcert.crt
+
+# actualy do the compilation
```
Comment (duilio) [inline Dockerfile]: typo: actually

File: pom.xml [hunk 2]
```diff
@@ -366,11 +366,9 @@
                             <goal>exec</goal>
                         </goals>
                         <configuration>
-                            <executable>git</executable>
+                            <executable>echo</executable>
```
Comment (duilio) [inline pom.xml:369]: reason for this change?

File: test-run.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
```
Comment (duilio) [inline test-run.sh:2]: maybe a one-line doc would be useful

File: test-stage.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
```
Comment (duilio) [inline test-stage.sh:2]: same here

File: README.md [hunk 5]
```diff
@@ -114,32 +114,39 @@ Secor comes with a number of tools implementing interactions with the environmen
...
-java -ea -Dlog4j.configuration=log4j.prod.properties -Dconfig=secor.prod.backup.properties -cp "secor-0.1-SNAPSHOT.jar:lib/*" com.pinterest.secor.main.ProgressMonitorMain
+java -ea -Dlog4j.configuration=log4j.prod.properties -Dconfig=secor.prod.backup.properties -cp "secor-0.26.jar:lib/*" com.pinterest.secor.main.ProgressMonitorMain
 ```
 
 Set `monitoring.interval.seconds` to a value larger than 0 to run in a loop, exporting stats every `monitoring.interval.seconds` seconds.
 
+## Viralize's changes
+Changes on top of v0.26 from Pinterest:
+
+- `com.pinterest.secor.monitoring.VoidMetricCollector` has been introduced as an alternative to `com.pinterest.secor.monitoring.OstrichMetricCollector`
```
Comment (duilio) [inline README.md]: + added docker file

**Review by duilio — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (frnidito) [reply pom.xml:369]: otherwise we need git in the docker

File: test-stage.sh (hunk 4 above)
Comment (frnidito) [reply test-stage.sh:2]: done

File: test-run.sh (hunk 3 above)
Comment (frnidito) [reply test-run.sh:2]: done

File: Dockerfile (hunk 1 above)
Comment (frnidito) [reply Dockerfile]: done

File: README.md (hunk 5 above)
Comment (frnidito) [reply README.md]: done

**Pull request closed without merging**